web_monitoring/html_diff_render.py

```diff
@@ -681,6 +714,21 @@ def fixup_chunks(chunks):
 
     return result
 
+
+def clean_resource_url(element, other):
+    try:
+        match_element = Strict_url_rule.REGEX_RULES[Strict_url_rule.mode].search(element)
+        if match_element:
+            element = element[match_element.end():]
+        match_other = Strict_url_rule.REGEX_RULES[Strict_url_rule.mode].search(other)
+        if match_other:
+            other = other[match_other.end():]
+        return str.__eq__(element, other) or str.__eq__(element.lower(), other.lower())
```

Comment:
What if these elements contain multiple things that look like URLs in them, or contain the same URLs in different orders? Consider that an `<img>` elements `alt` attribute could contain a URL, or that it might have multiple URLs in its `srcset`, or that a refactor of the page might have caused `src` and `srcset` attributes to be in a different order!

All these considerations might be different for different elements, so one improvement here might be to make this function take the *URLs* to compare, rather than the elements, and leave extracting the URLs to the token class, which can be customized for different elements or types of content.